src/lib.rs

```diff
@@ -987,6 +987,17 @@ impl Element {
             .collect())
     }
 
+    /// Look up a CSS value for this element by name (color, width, etc)
+    ///
+    /// See <https://www.w3.org/TR/webdriver/#dfn-get-element-css-value>.
+    pub async fn css(&mut self, prop: &str) -> Result<Option<String>, error::CmdError> {
```

Comment:
Is there still a reason for the method to return `Option<String>`, or should that just be `String` now?

As for documentation, I'd like to see it explicitly document what the caller should expect to get back if the property is malformed or hasn't been set.